lib/Benchmark/Remote/template/parameter_set_extractor.template

```diff
@@ -22,7 +22,19 @@ $parameterSets = array();
 ob_start();
 
 foreach ($paramProviders as $paramProvider) {
-    $parameterSets[] = $benchmark->$paramProvider();
+    if (is_callable($paramProvider)) {
+        $parameterSets[] = call_user_func($paramProvider);
+        continue;
+    }
+
+    if (method_exists($class, $paramProvider)) {
```

Comment:
Should be the other way round - local method to take precedence over global function f.e.

Reply:
This check will give false positives for non-public methods: https://3v4l.org/j5kOr.

Reply:
I guess we could normalize it to `$className . '::' . $paramProvider` https://3v4l.org/YnBCN

Reply:
Yep, either `is_callable("$className::$paramProvider")` or `is_callable([$benchmark, $paramProvider])` will work.